### PR TITLE
Fixed gulpfile: postcss cssnext already includes autoprefixer

### PR DESCRIPTION
**Pull request by ifraixedes**
PR for #1

### CODE REVIEW

File: gulpfile.babel.js [hunk 1]
```diff
@@ -87,10 +87,14 @@ const AUTOPREFIXER_BROWSERS = [
...
 gulp.task('styles', ['css', 'sass']);
 gulp.task('css', () => {
+  let postcssOpts = {
+    autoprefixer: {
+      browsers: AUTOPREFIXER_BROWSERS
+    }
+  };
   let processors = [
-    autoprefixer(AUTOPREFIXER_BROWSERS),
     pcssnext,
```
Comment (MoOx) [inline gulpfile.babel.js]: postcss-cssnext accepts some options (and it also have the same `browsers` option).

So this should do the trick

``` js
pcssnext({ browsers: AUTOPREFIXER_BROWSERS }) // browsers option is propagated to autoprefixer
```

**Comment by ifraixedes** [comment]
@MoOx  I've updated it as mentioned.

I thought that postcss-cssnext received the options through gulp-poscss.

Thanks.

**Comment by MoOx** [comment]
Most plugins take options as they are registered. This way, it's almost the same way to invoke them for each runners.

File: gulpfile.babel.js [hunk 2]
```diff
@@ -87,10 +87,14 @@ const AUTOPREFIXER_BROWSERS = [
 
 gulp.task('styles', ['css', 'sass']);
 gulp.task('css', () => {
+  let postcssOpts = {
+    autoprefixer: {
+      browsers: AUTOPREFIXER_BROWSERS
+    }
+  };
```
Comment (MoOx) [inline gulpfile.babel.js]: I guess you can remove this block.

File: gulpfile.babel.js [hunk 3]
```diff
@@ -99,7 +103,7 @@ gulp.task('css', () => {
   ])
   .pipe($.newer('.tmp/styles'))
   .pipe($.sourcemaps.init())
-  .pipe($.postcss(processors))
+  .pipe($.postcss(processors, postcssOpts))
```
Comment (MoOx) [inline gulpfile.babel.js]: and this too.

**Comment by ifraixedes** [comment]
@MoOx ah OK, I see and also like to think in trying to keep the same way to invoke for each runner.

**Comment by ifraixedes** [comment]
I've used postcss and cssnext as @MoOx mentioned in https://github.com/ifraixedes/landing/ and the result to generate the GH pages have been satisfactory so I decided to merge this changes.

**Pull request merged**